packages/fiori/src/Timeline.js

```diff
@@ -136,6 +137,32 @@ class Timeline extends UI5Element {
 			}
 		}
 	}
+
+	_onkeydown(event) {
+		if (isTabNext(event)) {
+			if (!event.target.nameClickable || event.isMarked === "link") {
+				this._handleTabNextOrPrevious(event, isTabNext(event));
```

Comment:
**isTabNext(event)** in "this._handleTabNextOrPrevious(event, **isTabNext(event));"**
is always true  as it's already checked in **"if (isTabNext(event)) {"**